packages/hoppscotch-app/locales/tw.json

```
@@ -245,11 +245,11 @@
     "communication": "通訊",
     "log": "日誌",
     "message": "訊息",
-    "publish": "釋出",
+    "publish": "發佈",
     "subscribe": "訂閱",
     "topic": "主題",
     "topic_name": "主題名稱",
-    "topic_title": "釋出／訂閱主題",
+    "topic_title": "發佈／訂閱主題",
```

```suggestion
    "topic_title": "發布／訂閱主題",
```
Ref: https://terms.naer.edu.tw/detail/16267516/?index=7
[Android](https://cs.android.com/android/platform/superproject/+/master:packages/services/Car/service/res/values-zh-rTW/strings.xml;l=87)
[GNOME](https://gitlab.gnome.org/GNOME/evolution/-/blob/master/po/zh_TW.po#L22270)